src/app/common/subscriptions/SubscriptionDetails.vue

```diff
@@ -1,59 +1,66 @@
 <template>
   <div>
-    <div v-if="details.globalInstanceId || details.connectTime || details.disconnectTime">
+    <div v-if="details.globalInstanceId || details.controlPlaneInstanceId || details.connectTime || details.disconnectTime">
       <h5 class="overview-tertiary-title">
         General Information:
       </h5>
 
-      <ul>
-        <li v-if="details.globalInstanceId">
-          <strong>Global Instance ID:</strong>
-          <span>{{ details.globalInstanceId }}</span>
-        </li>
+      <DefinitionList>
+        <DefinitionListItem
+          v-if="details.globalInstanceId"
+          term="Global instance ID"
```

Comment:
Do you think we'll ever need to include HTML in a `<dt>`, say an `<em>` or `<span>`, or maybe a `<a href>`. I can imagine a feature that term could have a little (i) icon, clicking on which would take you to some documentation, or show a tooltip or something, wdyt?

Reply:
I initially had implemented `DefinitionListItem` to take both a `term` and a `details` slot for the flexibility. I then realized that we never had a case where that was necessary for term: it was always a plain text node without any special-cased formatting. So then I simplified the component to accept a default slot instead of the two named slots because it really cuts down on markup used to use the component.

One default slot:

```vue
<DefinitionListItem term="Global instance ID">
  {{ details.globalInstanceId }}
</DefinitionListItem>
```

Two named slots:

```vue
<DefinitionListItem>
  <template #term>
    Global instance ID
  </template>

  <template #details>
    {{ details.globalInstanceId }}
  </template>
</DefinitionListItem>
```

I think the use case that additional stuff becomes necessary to be added is much more likely for the details part of the definition list so for now I prefer the simplification in use over the long-term flexibility.